bytes/bytes.mbt

```diff
@@ -29,6 +29,32 @@ pub fn Bytes::from_array(arr : Array[Byte]) -> Bytes {
   rv
 }
 
+///|
+/// Makes a new Bytes from a fixedarray.
+pub fn Bytes::from_fixedarray(
+  arr : FixedArray[Byte],
+  len~ : Int = arr.length()
```

Comment:
let's not use this feature, we may remove the labelled dependency to keep the language simple